PostHog/PostHogStorage.swift

```diff
@@ -75,16 +95,30 @@ class PostHogStorage {
         case sessionReplay = "posthog.sessionReplay"
         case isIdentified = "posthog.isIdentified"
         case personProcessingEnabled = "posthog.enabledPersonProcessing"
+
+        var useFileStorage: Bool {
+            #if os(tvOS)
+                [.replayQeueue, .oldQeueue, .queue].contains(self)
```

Comment:
i think more things should be here then because they can be heavy for user defaults (limit 500kb) eg flags, flags payloads, registerProperties, groups, we just cannot predict how much data would be because it really depends on how many flags and payloads they have ,etc